test/helpers/setup-helper.js

```diff
@@ -89,10 +91,35 @@ window.document.body.appendChild(popoverContent);
 // eslint-disable-next-line no-shadow
 const { default: fetch, Headers, Request, Response } = require('node-fetch');
 
-Object.assign(window, { fetch, Headers, Request, Response });
+const shimmedFetch = (url, ...args) => {
+  return fetch(url, ...args).catch((error) => {
+    if (error.message === 'Only absolute URLs are supported') {
+      const regex = /_locales\/([^/]+)\/messages\.json/gu;
+      const localeRelativePathRequest = url.match(regex);
+      if (localeRelativePathRequest && localeRelativePathRequest.length > 0) {
```

Comment:
Minor, but could this be simplified to `if(localeRelativePathRequest?.length)` ?